cli/command/service/create.go

```diff
@@ -73,6 +77,8 @@ func runCreate(dockerCli command.Cli, flags *pflag.FlagSet, opts *serviceOptions
 		return err
 	}
 
+	fmt.Printf("%v\n", service.TaskTemplate.Resources)
```

Comment:
This looks like debugging, please remove

Reply:
looks like this is still here

Reply:
Fixed...
I actually fixed that 4 times and merged that in the first commit.
Took me some time to understand that it was added in a later commit, rendering my fix useless.